banzai_floyds/settings.py

```diff
@@ -5,15 +5,18 @@
     'banzai.trim.Trimmer',
     'banzai.gain.GainNormalizer',
     'banzai.uncertainty.PoissonInitializer',
+    'banzai.cosmic.CosmicRayDetector',
```

Comment:
I've nominally added cosmic ray detection but we don't do much with that information yet.